static/app/views/insights/crons/components/onboarding.tsx

```diff
@@ -24,6 +28,35 @@ export function MonitorOnboarding({monitor}: Props) {
         )}
       </p>
       <MonitorQuickStartGuide monitor={monitor} />
+      <WaitingNotice>
+        <WaitingIndicator />
+        {t('Waiting for first Check-in')}
+        <WaitingHelpText>
+          {t(
+            'This Cron Monitor will not detect misses until the first Check-in has been received.'
+          )}
+        </WaitingHelpText>
+      </WaitingNotice>
     </OnboardingPanel>
   );
 }
+
+const WaitingNotice = styled('div')`
+  display: grid;
+  grid-template-columns: max-content 1fr;
+  align-items: center;
+  gap: ${space(0.25)} ${space(0.5)};
+  margin-top: ${space(2)};
+  color: ${p => p.theme.pink300};
```

Comment:
We’ll probably want to create a reusable component for all these onboarding waiting states at some point. But in the meantime, shouldn't this use `pink400` to match the color used in traces onboarding? see -> https://github.com/getsentry/sentry/blob/317b74889ae6fc61670c53a3f83fd8479705ad3b/static/app/views/performance/onboarding.tsx#L766